Weather/screens/YesterdayScreen.tsx

```diff
@@ -2,7 +2,8 @@ import { StyleSheet, Text, TouchableOpacity, View, Image } from 'react-native';
 import React, {useState, useEffect} from 'react';
 import {getWeatherStackKey} from '../App';
 
-const WEATHERSTACK_KEY = getWeatherStackKey();
+// const WEATHERSTACK_KEY = getWeatherStackKey();
```

Comment:
Fine to comment this out for now and hard code the key for our own use. Same as above, remove the key from the commit, and paste it in for our own testing.